### ISSUE TITLE
Preserve empty alt attribute value when minifying HTML

### ISSUE THREAD

**Issue by maxhoffmann** [wilsonzlin/minify-html#55]
## Expected Behavior
Minifying HTML this library should preserve `alt=""` attributes, so that screen readers know that images are decorative elements.
 
## Current Behavior
It currently minifies `alt=""` attributes to just `alt`, so that screen readers (and Google Page Speed) think that alternate text is missing.

## Step to reproduce
Minify HTML that includes an image tag, eg `<img src="some.png" alt="">`.

_(originally reported here: https://github.com/getzola/zola/issues/1683)_

**Comment by wilsonzlin** [comment]
Currently the minifier removes empty `alt=""` attributes completely, which is incorrect and has been fixed in `0.7.1`. Thanks for raising the issue.

`alt` and `alt=""` should be equivalent [according to the HTML5 specification](https://html.spec.whatwg.org/#a-quick-introduction-to-html), which is why the minifier removes the value part (`=""`). I did a quick test using a page with `<img src=".." alt>`:
- It passed the Google Lighthouse `Image elements have [alt] attributes` audit.
- macOS VoiceOver skipped the image when it had `alt` or `alt=""`, read the `alt` text when it had a nonempty value, and read the `src` value when the entire `alt` attribute was omitted.

Some [other](https://www.drupal.org/project/panopoly/issues/2905972#comment-12266339) [resources](https://stackoverflow.com/a/7731087/6249022) seem to also imply `alt` without a value works. So it seems that `alt=""` is only required for non-HTML5 conformity, which is outside the scope of this project. However I'm OK with adding an option that keeps `=""` for accessibility tags if there's interest.

**Comment by maxhoffmann** [comment]
Awesome thank you 🙂
